.pre-commit-config.yaml

```diff
@@ -30,6 +30,6 @@ repos:
     hooks:
       - id: flake8
   - repo: https://github.com/codespell-project/codespell
-    rev: v2.0.0
+    rev: v2.1.0
```

Comment:
I think this update to codespell receives an update to its dictionary to correct `statics -> statistics` which is why it wasn't correcting this before but this is an *incorrect* suggestion. We would have to add this word to a file of words to ignore and configure the hook to use that file... or rephrase the sentence.

Reply:
The one-sentence description of the docstring was grammatically incorrect anyways. I've pushed a commit to fix it. In particular, "statics" is now "static", which resolves the pipeline failure.